build-script.py

```diff
@@ -717,6 +794,15 @@ def main():
                 verbose=args.verbose,
                 add_source_locations=args.add_source_locations,
             )
+
+            run_code_generation(
```

Comment:
Added code generation here, as I think it makes sense to run this if the gyb have changed.

Reply:
I don’t think generation of files in SwiftSyntaxGeneration should be an implicit step in `build-script.py` because otherwise you might run into bootstrapping issues. Imagine you are making a breaking change to `SwiftSyntaxBuilder`. What you need to do is:
1. Change `SwiftSyntaxBuilderGeneration` to generate the code with the breaking change
2. Run `SwiftSyntaxBuilderGeneration` to actually generate the `SwiftSyntaxBuilder` code that contains the breaking change.
3. Update `SwiftSyntaxBuilderGeneration` to adopt the breaking change
4. Run step 1 and 2 again to make sure things are compatible
5. Actually build all the real SwiftSyntax products

Step (3) requires manual user intervention so we can’t just perform the other steps automatically in build-script.py.

What I would imagine is that there is a mode for build-script.py that just build + runs SwiftSyntaxBuilderGeneration.